src/components/Player/Header/PlayerHeader.jsx

```diff
@@ -188,6 +189,11 @@ grid-template-columns: 1fr minmax(min-content, ${constants.appWidth}px) 1fr;
   }
 }
 
+.lockIcon {
```

Comment:
This also isn't needed?

Reply:
yeah so sorry my bad, have removed the unnecessary code and have double checked again if i left out anything else